tests/gordo_components/builder/test_mlflow_utils.py

```diff
@@ -14,143 +15,213 @@
 def metadata():
     """Example metadata for testing"""
     return {
-        "user-defined": {},
-        "name": "model-name",
+        "user-defined": {
+            "global-metadata": {},
+            "machine-metadata": {
+                "information": "Some sweet information about the model"
+            },
+        },
+        "name": "crazy-sweet-name",
         "dataset": {
             "tag_list": [
-                SensorTag(name="Tag 1", asset=None),
-                SensorTag(name="Tag 2", asset=None),
+                SensorTag(name="Tag-1", asset="asgb"),
+                SensorTag(name="Tag-2", asset="asgb"),
             ],
             "target_tag_list": [
-                SensorTag(name="Tag 1", asset=None),
-                SensorTag(name="Tag 2", asset=None),
+                SensorTag(name="Tag-3", asset="asgb"),
+                SensorTag(name="Tag-4", asset="asgb"),
             ],
-            "train_start_date": datetime(2017, 12, 25, 6, 0, tzinfo=UTC),
-            "train_end_date": datetime(2017, 12, 30, 6, 0, tzinfo=UTC),
+            "train_start_date": datetime.datetime(2019, 1, 1, 0, 0, tzinfo=tzutc()),
+            "train_end_date": datetime.datetime(2019, 3, 1, 0, 0, tzinfo=tzutc()),
             "resolution": "10T",
             "filter": "",
             "row_filter_buffer_size": 0,
             "data_provider": {"type": "RandomDataProvider"},
-            "asset": None,
+            "asset": "asgb",
         },
         "model": {
             "model-offset": 0,
-            "model-creation-date": "2019-11-19 10:55:19.327827+01:00",
-            "model-builder-version": "0.39.1.dev39+gee79144.d20191118",
+            "model-creation-date": "2019-12-13 07:16:26.482309+01:00",
+            "model-builder-version": "0.43.1.dev14+g50d5a9e",
             "model-config": {
-                "sklearn.compose.TransformedTargetRegressor": {
-                    "regressor": {
+                "gordo_components.model.anomaly.diff.DiffBasedAnomalyDetector": {
+                    "base_estimator": {
                         "sklearn.pipeline.Pipeline": {
                             "steps": [
-                                "sklearn.preprocessing.data.MinMaxScaler",
+                                "sklearn.decomposition.pca.PCA",
                                 {
-                                    "gordo_components.model.models.KerasAutoEncoder": {
-                                        "kind": "feedforward_hourglass",
-                                        "compression_factor": 0.5,
-                                        "encoding_layers": 2,
-                                        "func": "tanh",
-                                        "out_func": "linear",
-                                        "epochs": 3,
+                                    "sklearn.multioutput.MultiOutputRegressor": {
+                                        "estimator": "sklearn.linear_model.base.LinearRegression"
```

Comment:
I guess all this is a copy-paste update?

Reply:
Correct.

Reply:
Is it needed? Sorry, I'm just a bit skeptical of this mlflow utils test stuff.. Are we just copying and pasting because the tests would fail otherwise or?